docs/GettingStarted.md

```diff
@@ -81,21 +81,35 @@ jest --init
 
 ### Using Babel
 
-To use [Babel](http://babeljs.io/), install the `babel-jest` and `@babel/core` packages via `yarn`:
+To use [Babel](http://babeljs.io/), install required development dependencies via `yarn`:
 
 ```bash
-yarn add --dev babel-jest @babel/core
+yarn add --dev @babel/core @babel/preset-env @babel/preset-react
 ```
 
-Don't forget to add a [`babel.config.js`](https://babeljs.io/docs/en/config-files) file in your project's root folder. For example, if you are using ES2018 and [React](https://reactjs.org) with the [`@babel/preset-env`](https://babeljs.io/docs/en/babel-preset-env) and [`@babel/preset-react`](https://babeljs.io/docs/en/babel-preset-react) presets:
+Install `@babel/polyfill`:
 
-```js
-module.exports = {
-  presets: ['@babel/preset-env', '@babel/preset-react'],
-};
+```bash
+yarn add @babel/polyfill
 ```
 
-You are now set up to use all ES2018 features and React-specific syntax.
+Load the polyfill and configure Babel in your `package.json` file:
```

Comment:
no, we should recommend the config file

Reply:
How come we would recommend configuring Jest in `package.json` but not Babel?

Reply:
IMO we should recommend using `jest.config.js` as well.

The reason we want to recommend a `babel.config.js` though is because that's what babel itself does: https://babeljs.io/docs/en/configuration

Reply:
Got it, I changed the note to use `jest.config.js`.